### PR TITLE
Add Recursive Compression to OPMRUN Tools

### PR DESCRIPTION
**Pull request by OPMUSER**
Add recursive compression to compress all *.DATA jobs in a directory and below, as well as  ensemble recursive compression, that will compress all files in an ensemble directory. I needed this for a project that had far too many ensemble cases.

Added additional documentation and compression "help" option.

Fixed a bug with calling OPM ResInsight.

This was done some time ago I just needed to update the image in the manual and the web site for the commit.

### CODE REVIEW

**Comment by OPMUSER** [comment]
Ready for review and hopefully merge.

File: opmrun/opm_compress.py [hunk 1]
```diff
@@ -247,6 +315,54 @@ def compress_files(opmoptn):
...
+    Returns
+    ------
+    None
+    """
+
+    helptext = (
+                'OPMRun is a Graphical User Interface ("GUI") program for the Open Porous Media ("OPM") Flow ' +
+                'simulator. \n' +
+                '\n'
+                'The Compress and Uncompress utilities enable compress and uncompression to be performed on OPM Flow ' +
```
[the remark this answers is not in the document]
Comment (OPMUSER) [reply opmrun/opm_compress.py]: Done.

File: opmrun/opm_compress.py [hunk 2]
```diff
@@ -229,12 +291,18 @@ def compress_files(opmoptn):
...
-                sg.cprint('Start Compression\n')
+                sg.cprint('\nStart Compression')
                 (job, jobcmd, jobpath, jobbase, jobroot, jobfile, jobzip) = compress_cmd(cmd)
-
-                error = change_directory(jobpath, popup=True, outprt=True)
-                if not error:
+                #
+                # File or Ensemble Compression
+                #
+                if Path(cmd).is_file():
```
[the remark this answers is not in the document]
Comment (OPMUSER) [reply opmrun/opm_compress.py:299]:
```
Path.is_file()

    Return True if the path points to a regular file (or a symbolic link pointing to a regular file), False if it points to another kind of file.

    False is also returned if the path doesn’t exist or is a broken symlink; other errors (such as permission errors) are propagated.

```

[the remark this answers is not in the document]
Comment (blattms) [reply opmrun/opm_compress.py:299]: exactly. I was wondering what happens in the else branch in this case. It assumes a directory but gets a symlink instead.

**Pull request merged**